tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -3292,28 +3292,17 @@ class HLSLExternalSource : public ExternalSemaSource {
           break;
```

Comment:
Eventually I'd like to migrate the `CreateSubobject***` functions above to using my new Builder class as well but I chose not to go there in this change.